dev-docs/examples/basic-example.md

```diff
@@ -16,7 +16,12 @@ pid: 10
 
 ## Basic Prebid.js Example
 
-{% capture htmlCodePrebid %}<h5>Div-1</h5>
+{% capture htmlCodePrebid %}
+<!-- DO NOT add this script in prod -->
+<script async src="intercept-banner-not-for-prod.js" ></script>
```

Comment:
```suggestion
<script src="intercept-banner-not-for-prod.js" ></script>
```

This should probably be not `async` to avoid race conditions with the script tag below that fires the requests